src/nvim/tui/tui.c

```diff
@@ -1535,7 +1540,8 @@ static void patch_terminfo_bugs(TUIData *data, const char *term,
   bool rxvt = terminfo_is_term_family(term, "rxvt");
   bool teraterm = terminfo_is_term_family(term, "teraterm");
   bool putty = terminfo_is_term_family(term, "putty");
-  bool screen = terminfo_is_term_family(term, "screen");
+  bool screen = terminfo_is_term_family(term, "screen") && !os_getenv("TMUX");
```

Comment:
Note to myself: I got surprised by this thinking tmux was recommending TERM=tmux but the FAQ states otherwise and TERM=screen is fine too https://github.com/tmux/tmux/wiki/FAQ